manifests/pnp4nagios/config.pp

```diff
@@ -0,0 +1,75 @@
+class icinga2::pnp4nagios::config inherits icinga2::pnp4nagios {
+
+  $monitoring_type = 'icinga'
+  $log_type = 'syslog'
+  $debug_lvl = 1
+
+  file { "$httpd_path/$htpasswd_config":
+    ensure => $htpasswd_config_ensure,
+    owner  => $htpasswd_user,
+    group  => $htpasswd_group,
+    mode   => $htpasswd_config_mode,
+    source => "puppet:///modules/${module_name}/pnp/$htpasswd_config",
+    notify => Service['httpd'],
+  }
+
+  file { '/etc/nagios/passwd':
+    ensure  => file,
+    owner   => $htpasswd_user,
+    group   => $htpasswd_group,
+    notify  => Service['httpd'],
+  }
+
+  exec { 'htpasswd':
+    path    => '/bin:/usr/bin:/sbin:/usr/sbin',
+    command => "htpasswd -db /etc/nagios/passwd $nagios_web_user $nagios_web_pass",
+    subscribe   => File["/etc/nagios/passwd"],
+    refreshonly => true
+  }
+
+  file { '/etc/pnp4nagios/npcd.cfg':
+    owner => $htpasswd_user,
+    group => $htpasswd_group,
+    notify => Class[icinga2::pnp4nagios::service],
+    content => template('icinga2/npcd.cfg.erb'),
+  }
+
+  file_line{ 'date.timezone':
```

Comment:
Please make a class parameter called $manage_php_timezone and check here if that is true. This enables people to tell the module not to manage php's timezone as they might not want to.